airbyte-webapp/src/components/EntityTable/ConnectionTable.tsx

```diff
@@ -25,6 +27,16 @@ interface IProps {
   onSync: (id: string) => void;
 }
 
+type ColumnDefs = [
+  ColumnDef<ITableDataItem, string>,
+  ColumnDef<ITableDataItem, string>,
+  ColumnDef<ITableDataItem, string>,
+  ColumnDef<ITableDataItem, ConnectionScheduleData>,
+  ColumnDef<ITableDataItem, number>,
+  ColumnDef<ITableDataItem, boolean>,
+  ColumnDef<ITableDataItem, string>
+];
+
```

Comment:
Please, use `columnHelper()` from `@tanstack/react-table` so we don't need to redefine types by ourselves.
Doc: https://tanstack.com/table/v8/docs/guide/column-defs#column-helpers